### PR TITLE
Use Kontena::Command hooks and new authentication mechanism.

### PR DESCRIPTION
**Pull request by kke**

### CODE REVIEW

File: lib/kontena/machine/aws/cloudinit_master.yml [hunk 1]
```diff
@@ -89,8 +89,11 @@ coreos:
             --link kontena-server-mongo:mongodb \
             -e MONGODB_URI=mongodb://mongodb:27017/kontena_server \
   <% end -%>
-  <% if auth_server %>
-            -e AUTH_API_URL=<%= auth_server %> \
+  <% if server_name %>
```
Comment (jakolehm) [inline lib/kontena/machine/aws/cloudinit_master.yml]: should be `<% if server_name -%>` (no newlines allowed in this config)... check also other conditions.

File: lib/kontena/machine/aws/cloudinit_master.yml [hunk 2]
```diff
@@ -89,8 +89,11 @@ coreos:
...
+            -e KONTENA_SERVER_NAME=<%= server_name %> \
```
Comment (jakolehm) [inline lib/kontena/machine/aws/cloudinit_master.yml]: should be `SERVER_NAME`

File: lib/kontena/machine/aws/cloudinit_master.yml [hunk 3]
```diff
@@ -89,8 +89,11 @@ coreos:
...
+  <% end -%>
+  <% if initial_admin_code %>
+            -e KONTENA_INITIAL_ADMIN_CODE=<%= intial_admin_code %> \
```
Comment (jakolehm) [inline lib/kontena/machine/aws/cloudinit_master.yml]: should be `INITIAL_ADMIN_CODE`

File: lib/kontena/machine/aws/master_provisioner.rb [hunk 4]
```diff
@@ -99,13 +100,18 @@ def run!(opts)
...
-          sleep 5 until master_running?(http_client)
+        spinner "Waiting for #{name.colorize(:cyan)} to start " do
+          sleep 1 until master_running?(http_client)
         end
       end
-
-      puts "Kontena Master is now running at #{master_url}"
-      puts "Use #{"kontena login --name=#{name.sub('kontena-master-', '')} #{master_url}".colorize(:light_black)} to complete Kontena Master setup"
+      puts
+      puts "Kontena Master is now running at #{master_url}".colorize(:green)
```
Comment (jakolehm) [inline lib/kontena/machine/aws/master_provisioner.rb]: These look funny in the output, maybe fake spinner or something is better?

**Comment by kke** [comment]
Fixes applied

**Pull request merged**